src/components/BoardNewColumn/BoardNewColumn.tsx

```diff
@@ -0,0 +1,15 @@
+import './BoardNewColumn.css';
+
+interface BoardNewColumnProps {
+  onClick?: () => void;
+}
+
+function BoardNewColumn({ onClick }: BoardNewColumnProps) {
+  return (
+    <button className="new-column-btn" onClick={onClick}>
```

Comment:
Es ideal usar el nombre del componente como nombre principal de la clase
```suggestion
    <button className="board-new-column" onClick={onClick}>
```